notebooks/Isha-AnomalyDetection.ipynb

```diff
@@ -0,0 +1,1452 @@
+{
```

Comment:
do a PR FROM `gsoc-dev`, with arrow pointed at your branch to clean these notebooks up, @abhinavbajpai2012